### DISCUSSION TITLE
Deployment Failed

### DISCUSSION THREAD

**Discussion by sabljakd** [vercel/vercel#5170] [Help]
. After, I connected my Vercel account to my Gitlab account. Then I tried to deploy. After waiting for 2 minutes, it said Deployment Failed. It doesn't appear to give any reason why. What am I to do? 

Build Failed
Command "yarn run build" exited with 1

LOGS:

18:16:30.130  	Cloning github.com/sabljakd/FoodsetHSN (Branch: master, Commit: 1336684)
18:16:30.760  	Cloning completed in 630ms
18:16:30.762  	Analyzing source code...
18:16:31.347  	Installing build runtime...
18:16:31.763  	Build runtime installed: 416.107ms
18:16:32.175  	Looking up build cache...
18:16:32.213  	Build cache not found
18:16:32.633  	Installing dependencies...
18:16:32.865  	yarn install v1.22.4
18:16:32.940  	[1/4] Resolving packages...
18:16:34.161  	warning node-sass > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
18:16:34.235  	warning node-sass > request > har-validator@5.1.5: this library is no longer supported
18:16:34.431  	warning popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
18:16:36.187  	[2/4] Fetching packages...
18:18:14.949  	info fsevents@1.2.11: The platform "linux" is incompatible with this module.
18:18:14.950  	info "fsevents@1.2.11" is an optional dependency and failed compatibility check. Excluding it from installation.
18:18:14.961  	info fsevents@2.1.2: The platform "linux" is incompatible with this module.
18:18:14.961  	info "fsevents@2.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
18:18:14.962  	[3/4] Linking dependencies...
18:18:25.022  	[4/4] Building fresh packages...
18:18:27.566  	success Saved lockfile.
18:18:27.570  	Done in 114.71s.
18:18:27.838  	Running "yarn run build"
18:18:28.014  	yarn run v1.22.4
18:18:28.037  	$ vue-cli-service build
18:18:28.498  	-  Building for production...
18:18:30.144  	 ERROR  Failed to compile with 1 errors4:18:30 PM
18:18:30.144  	 error  in ./src/main.js
18:18:30.144  	Module build failed (from ./node_modules/@vue/cli-plugin-babel/node_modules/thread-loader/dist/cjs.js):
18:18:30.144  	Thread Loader (Worker 0)
18:18:30.145  	[BABEL] /vercel/72cb1d2c/src/main.js: No "exports" main resolved in /vercel/72cb1d2c/node_modules/@vue/cli-plugin-babel/node_modules/@babel/helper-compilation-targets/package.json (While processing: "/vercel/72cb1d2c/node_modules/@vue/cli-plugin-babel/preset.js")
18:18:30.145  	    at PoolWorker.fromErrorObj (/vercel/72cb1d2c/node_modules/@vue/cli-plugin-babel/node_modules/thread-loader/dist/WorkerPool.js:262:12)
18:18:30.145  	    at /vercel/72cb1d2c/node_modules/@vue/cli-plugin-babel/node_modules/thread-loader/dist/WorkerPool.js:204:29
18:18:30.145  	    at applyExports (internal/modules/cjs/loader.js:490:9)
18:18:30.145  	    at resolveExports (internal/modules/cjs/loader.js:506:23)
18:18:30.145  	    at Function.Module._findPath (internal/modules/cjs/loader.js:634:31)
18:18:30.145  	    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:952:27)
18:18:30.145  	    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
18:18:30.145  	    at Module.require (internal/modules/cjs/loader.js:1025:19)
18:18:30.145  	    at require (internal/modules/cjs/helpers.js:72:18)
18:18:30.145  	    at Object.<anonymous> (/vercel/72cb1d2c/node_modules/@vue/cli-plugin-babel/node_modules/@babel/preset-env/lib/debug.js:8:33)
18:18:30.145  	    at Module._compile (internal/modules/cjs/loader.js:1137:30)
18:18:30.146  	    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
18:18:30.146  	 @ multi ./src/main.js
18:18:30.146  	 ERROR  Build failed with errors.
18:18:30.178  	error Command failed with exit code 1.
18:18:30.178  	info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
18:18:30.186  	Error: Command "yarn run build" exited with 1
18:18:32.227  	Done with "package.json"

**Comment by paulogdm** [comment]
Hello,

Please get in touch with support: https://vercel.com/support/request

**Reply by dreampulse** [reply]
I've the same issue!!!
